dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/Event.java

```diff
@@ -543,6 +547,28 @@ public void setAssignedUserDisplayName( String assignedUserDisplayName )
         this.assignedUserDisplayName = assignedUserDisplayName;
     }
 
+    @JsonIgnore
```

Comment:
are you doing this so its not showing up in the "old" export?

Reply:
Yes, I needed those fields in the object but I don't want to change the old importer